### ISSUE TITLE
Issue 1: Alternating Turns 

### ISSUE THREAD

**Issue by Joy-ctrl** [Joy-ctrl/Joy-s-Computer-Science#2]
I'm trying to make it so the first encounter the pokemon have is based off speed, and then after that it just alternates turns. But after selecting my first attack the code ends.
This is the code:

```
    print("You're walking through a forest... when a wild Bulbasaur appears!")
    time.sleep(1)
    battle_turn = 1
    if battle_turn == 1:
        print("You have a higher speed, so you attack first!")
    attack1 = input("Pick an attack:\n1 - Tackle\n2 - Water Gun\n3 - Bubble Beam\n ")
    if int(attack1) == 1: 
        print("You tackled the wild Bulbasaur")
        time.sleep(1)
    elif int(attack1) == 2:
        print("You performed Water Gun on the wild Bulbasaur")
        time.sleep(1)
    elif int(attack1) == 3:
        print("You blinded the wild Bulbasaur with Bubble Beam")
        Bhealth = Bhealth - Sattack + Bdefence
        time.sleep(1)
    else:
        print("Sorry that didn't work please try again")
    exit()
    if Bspeed > Sspeed:
        print("Bulbasaur has a higher speed, so they attack first!")
        print("Bulbasaur uses Vine Whip")
    Shealth = Shealth - Battack + Sdefence
    if Shealth > 0 and Bhealth <= 0:
        print("The winner is Squirtle!")
    if Bhealth > 0 and Shealth <= 0:
        print("The winner is Bulbasaur")
```

**Comment by markcurtis1970** [comment]
Ok so your code has incorrect indents it seems, put this into `homework.py` then update this issue with the example output you  see when you run the code, cheers!

**Comment by markcurtis1970** [comment]
Fixed by  commit e91718a74dcf3068d6849ffe7a059420198d2fb5
